assets/src/settings-page/plugin-suppression.js

```diff
@@ -0,0 +1,46 @@
+
+/**
+ * WordPress dependencies
+ */
+import { useContext, useEffect, useRef, useState } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { Options } from '../components/options-context-provider';
+import { Selectable } from '../components/selectable';
+
+/**
+ * Plugin suppression section of the settings page.
+ */
+export function PluginSuppression() {
+	const { fetchingOptions } = useContext( Options );
+
+	const [ supportedTemplatesTableFound, setSupportedTemplatesTableFound ] = useState( false );
+
+	const pluginSuppressionContainer = useRef();
+
+	/**
+	 * Pull the PHP-generated plugin suppression section into the JS-generated area.
+	 */
+	useEffect( () => {
+		const settingsSections = [ ...document.querySelectorAll( '#amp-settings-sections > table' ) ];
+		const supportedTemplatesTable = settingsSections.find( ( section ) => section.querySelector( '.amp-suppressed-plugins' ) );
+		if ( supportedTemplatesTable ) {
+			pluginSuppressionContainer.current.appendChild( supportedTemplatesTable );
+			setSupportedTemplatesTableFound( true );
+		}
+	}, [] );
+
+	return (
+		<section className={ fetchingOptions || ! supportedTemplatesTableFound ? 'hidden' : '' }>
+			<h2>
+				{ __( 'Plugin Suppression', 'amp' ) }
+			</h2>
+			<Selectable className="plugin-suppression">
+				<div ref={ pluginSuppressionContainer } />
```

Comment:
The status column is not wide enough in this context:

![image](https://user-images.githubusercontent.com/134745/87075225-db053300-c1d4-11ea-8e1b-7d66905298bc.png)

Reply:
Updated in d9bab74c0 
<img width="395" alt="Screen Shot 2020-07-09 at 2 03 22 PM" src="https://user-images.githubusercontent.com/9020968/87080330-17dd2400-c1ed-11ea-8e35-c30e723f68d6.png">